From ad799aa8244746b35610216578cba4ace17e7b6a Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Mon, 27 Jul 2015 16:28:48 +0800
Subject: [PATCH 0019/5242] MLK-10369 ARM: dts: Add i2c bus support for
 i.MX7d-sdb board

commit  a15af6c72a5831dd4ddae592c6169857f35938a2 from
https://source.codeaurora.org/external/imx/linux-imx.git

- Add i2c1,2,3,4 bus support for i.MX7d-sdb board (i2c4 bus need to
  rework: swap R485 and R33).
- Add i2c device PMIC support.

Signed-off-by: Fugang Duan <B38611@freescale.com>
(cherry picked from commit: 45acb911f7192fcc574f1e0faaf19513b9879046)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 8cc0a12..7752692 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -219,6 +219,7 @@
 };
 
 &i2c1 {
+	clock-frequency = <100000>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_i2c1>;
 	status = "okay";
@@ -315,6 +316,7 @@
 };
 
 &i2c2 {
+	clock-frequency = <100000>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_i2c2>;
 	status = "okay";
@@ -326,12 +328,14 @@
 };
 
 &i2c3 {
+	clock-frequency = <100000>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_i2c3>;
 	status = "okay";
 };
 
 &i2c4 {
+	clock-frequency = <100000>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_i2c4>;
 	status = "okay";
-- 
1.7.9.5

